src/app/util/attribute-metadata.h

```diff
@@ -100,7 +100,7 @@ typedef struct
 union EmberAfDefaultOrMinMaxAttributeValue
 {
     constexpr EmberAfDefaultOrMinMaxAttributeValue(const uint8_t * ptr) : ptrToDefaultValue(ptr) {}
-    constexpr EmberAfDefaultOrMinMaxAttributeValue(uint16_t val) : defaultValue(val) {}
+    constexpr EmberAfDefaultOrMinMaxAttributeValue(uint32_t val) : defaultValue(val) {}
```

Comment:
I could still allow uint16_t constructor arguments to make sure we don't break tests outside of matter repo. Right now I am not sure if that is important to do or not

Reply:
I think we should be ok with changing this. External tests likely just use constants and u16 is trivially extended to u32 without any data loss.